Jenkinsfile

```diff
@@ -67,14 +67,19 @@ pipeline {
         stage('Build OptaPlanner') {
             steps {
                 script {
-                    getMavenCommand(optaplannerRepo)
-                        .withProfiles(['run-code-coverage'])
+                    mvnCmd = getMavenCommand(optaplannerRepo, true, true)
                         .withProperty('full')
-                        .run('clean install')
+                    if(!isSpecificPRCheck()) {
```

Comment:
Nitpicking, but would you mind changing the logic and call the method `isNormalPRCheck()`? I think it reads slightly more naturally.

Reply:
changed